src/ansys/workbench/core/public_api.py

```diff
@@ -118,6 +118,8 @@ def __init__(
         self._wmi_connection = None
         self._process_id = -1
 
+        if not version:
```

Comment:
why not to keep the default argument value?

Reply:
It is one of the requirements from ACE to use None as default, to be consistent with other PyAnsys products.